src/tools/build-manifest/src/versions.rs

```diff
@@ -106,6 +108,8 @@ impl PkgType {
             ReproducibleArtifacts => HOSTS,
             RustcDocs => HOSTS,
             Cargo => HOSTS,
+            // FIXME should this use the exact list of targets for which we build cg_clif?
+            RustcCodegenCranelift => HOSTS,
```

Comment:
No, I don't think so. The component might be available on any host platform, it'll get marked as unavailable automatically in the generated toml IIRC. I'm not sure there's any real impact from extra platforms getting listed.